### PR TITLE
Port documentation from the wiki: Boolean field

### PR DESCRIPTION
**Pull request by kn-neelalohitha**
Signed-off-by: knn <kn.neelalohitha@gmail.com>
# What is this PR/ Whe we need it:
This PR ports the [Boolean Field](http://wiki.sensenet.com/Boolean_Field) documentation from wiki to the [_docs](https://github.com/SenseNet/sensenet.github.io/tree/master/_docs) folder.
## Commit Summary:
- [x]  Add a header as specified in the issue #324 
- [x] Change links from old wiki to as specified in #324 
- [ ] ~Recreating old screenshots (Need help!)~

# Which issue this PR fixes:
#324

### CODE REVIEW

File: _docs/boolean-field.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+- the field will be _hidden_ in browse mode and it will be under the _advanced fields_ in edit and new mode when the Content is displayed with [Generic Content View](./Generic-Content-View).
+
+## Related links
+
+- [Field](./Field)
+- [Boolean Field Control](./Boolean-Field-Control)
+
+## References
+
+[Category:](./Category) [Fields](./Field)
```
Comment (herflis) [inline _docs/boolean-field.md]: you can remove this part

**Comment by herflis** [comment]
Hey @kn-neelalohitha ,
thank you for your contribution! 🙏 I've made a minor suggetion, otherwise it is ok. There's no screenshot in the old wiki article, so you don't have to add it. 🙂

**Comment by kn-neelalohitha** [comment]
@herflis Thanks for feedback!I have incorporated your suggestions!

**Comment by kn-neelalohitha** [comment]
> Hey @kn-neelalohitha ,
> thank you for your contribution! 🙏 I've made a minor suggetion, otherwise it is ok. There's no screenshot in the old wiki article, so you don't have to add it. 🙂

No problem!Always happy to contribute to open source and love to be a part of community!

**Pull request merged**